src/package/utils/lessonStates/index.js

```diff
@@ -17,25 +17,25 @@ export const detailsByLessonState = {
     action: 'Accept',
     description: 'This lesson has been accepted - it can be claimed.',
     color: 'green',
-    icon: 'check',
+    icon: 'add',
```

Comment:
Replace check icons on lesson actions with "+" for now to avoid confusion with the new instructor get published checklist check icons
https://github.com/eggheadio/egghead-ui/issues/142